mscore/preferences.cpp

```diff
@@ -995,15 +991,11 @@ void PreferenceDialog::updateValues()
       myPlugins->setText(prefs.myPluginsPath);
       mySoundfonts->setText(prefs.mySoundfontsPath);
 
-      idx = 0;
-      int n = sizeof(exportAudioSampleRates)/sizeof(*exportAudioSampleRates);
-      for (;idx < n; ++idx) {
-            if (exportAudioSampleRates[idx] == prefs.exportAudioSampleRate)
-                  break;
-            }
-      if (idx == n)     // if not found in table
-            idx = 0;
-      exportAudioSampleRate->setCurrentIndex(idx);
+      index = exportAudioSampleRate->findText(QString("%1").arg(prefs.exportAudioSampleRate));
```

Comment:
I will have another look but I guess these two need to be changed as well. If we save the text from the combo box in prefs.exportAudioSampleRate it will be translated. If the translation is different and the user changes his language, then when reading it back it will not work well. That's why other part of the code use the userData of the combobox item and not the text.